test/unit_tests/test_interrogate_setup.py

```diff
@@ -111,10 +111,34 @@ def test_generate_cmake_file_invalid(self):
                           modules=[])
 
     def test_interrogate_setup_py(self):
+        # setup python packages
+        curdir = os.getcwd()
+        workdir = tempfile.mkdtemp()
+        # src/foo/__init__.py package
+        os.makedirs(os.path.join(workdir, 'src/foo'))
+        open(os.path.join(workdir, 'src/foo/__init__.py'), 'w')
+        # foo/__init__.py, foo/bin/foo
+        # bar/__init__.py, bar/nodes/bar
+        os.mkdir(os.path.join(workdir, 'foo'))
+        open(os.path.join(workdir, 'foo/__init__.py'), 'w')
+        os.makedirs(os.path.join(workdir, 'foo/bin'))
+        open(os.path.join(workdir, 'foo/bin/foo'), 'w')
+        os.makedirs(os.path.join(workdir, 'bar'))
+        open(os.path.join(workdir, 'bar/__init__.py'), 'w')
+        os.makedirs(os.path.join(workdir, 'bar/nodes'))
+        open(os.path.join(workdir, 'bar/nodes/bar'), 'w')
+        # src/__init__.py, lib/__init__.py
+        open(os.path.join(workdir, 'src/__init__.py'), 'w')
+        os.makedirs(os.path.join(workdir, 'lib'))
+        open(os.path.join(workdir, 'lib/__init__.py'), 'w')
+        # change directory for testing
+        os.chdir(workdir)
```

Comment:
They are necessary to test various combinations of `package_dir` in below lines.

like:
```
fake_setup(version='0.1.1', package_dir={}, packages=['foo', 'bar'], scripts=['bin/foo', 'nodes/bar'])
fake_setup(version='0.1.1', package_dir={'foo': 'src', 'bar': 'lib'}, packages=['foo', 'bar'],)
```